### PR TITLE
[i18n] Update en-UK grammar and style

### PR DESCRIPTION
**Pull request by Frenzie**
Taken from <https://github.com/FreshRSS/FreshRSS/pull/3060>

- [x] clear commit messages
- [x] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

---
This change is [Reviewable](https://reviewable.io/reviews/freshrss/freshrss/3068)

### CODE REVIEW

**Comment by Frenzie** [comment]
Oh drat, can one of the flags to manipulate.translation be used to update all todos without getting rid of the todo @aledeg?

**Comment by aledeg** [comment]
@Frenzie What do you mean by that? I do not understand your question!

**Comment by Frenzie** [comment]
@aledeg See https://github.com/FreshRSS/FreshRSS/pull/3068/commits/96bca62bb535f82d3e97172b2f5719f61de54635 By updating the en-UK file there's a whole bunch of stuff I'll have to update manually to ensure it remains todo. It's not *too* much work, just something I rather wouldn't do if I don't have to. ;-)

**Review by Alkarex — APPROVED** [review]

**Review by Alkarex — APPROVED** [review]

**Comment by Frenzie** [comment]
@Alkarex Need to update quite a few more unfortunately. :-)

File: app/i18n/cz/user.php [hunk 1]
```diff
@@ -3,8 +3,8 @@
 return array(
 	'email' => array(
 		'feedback' => array(
-			'invalid' => 'The email address is invalid.',	// TODO - Translation
-			'required' => 'The email address is required.',	// TODO - Translation
```
Comment (aledeg) [inline app/i18n/cz/user.php:7]: That was bad!

File: app/i18n/ru/user.php [hunk 2]
```diff
@@ -3,19 +3,19 @@
...
 		'feedback' => array(
-			'invalid' => 'The email address is invalid.',	// TODO - Translation
-			'required' => 'The email address is required.',	// TODO - Translation
+			'invalid' => 'This email address is invalid.',	// TODO - Translation
+			'required' => 'An email address is required.',	// TODO - Translation
 		),
 		'validation' => array(
 			'change_email' => 'You can change your email address <a href="%s">on the profile page</a>.',	// TODO - Translation
-			'email_sent_to' => 'We sent you an email at <strong>%s</strong>, please follow its indications to validate your address.',	// TODO - Translation
+			'email_sent_to' => 'We sent you an email at <strong>%s</strong>, please follow its indications to validate your address.',
```
Comment (Frenzie) [inline app/i18n/ru/user.php]: drat, I accidentally committed this whole chunk

**Pull request merged**